### PR TITLE
Add a check that reaperId is never null or empty

### PR DESCRIPTION
**Pull request by mmatalka**
@Yarin78 @varjoranta

### CODE REVIEW

File: src/main/java/com/spotify/reaper/cassandra/JmxProxy.java [hunk 1]
```diff
@@ -471,6 +471,9 @@ public static Integer versionCompare(String str1, String str2) throws ReaperExce
   }
 
   public void clearSnapshot(String repairId, String keyspaceName) throws ReaperException {
+    if (repairId == null || repairId.equals("")) {
```
Comment (Yarin78) [inline src/main/java/com/spotify/reaper/cassandra/JmxProxy.java]: Very nitpicky, but maybe add a comment why we disallow this.

**Comment by mmatalka** [comment]
@Yarin78 Updated

**Comment by Yarin78** [comment]
+1

**Comment by varjoranta** [comment]
:+1:

**Pull request merged**